CaseStudy/Assignment-3/WebContent/Css/Search.css

```diff
@@ -0,0 +1,111 @@
+@CHARSET "ISO-8859-1";
+body {
+	margin: 0px;
+	background: url(../Resources/background.jpg) no-repeat center fixed;
+	background-color: grey;
+	background-size : cover;
+}
+header {
+	width: 100%;
+	height: 120px;
+	background: linear-gradient(to left,black,#3A3A3A, black);
+	padding-top: 0px;
+	border-bottom: solid 1px white;
+	z-index: 0;
+}
+
+.part {
+	margin-top: 70px;
+	float: right;
+	margin-right: 10%;
+}
+
+.part a, label {
+	color: white;
+	font-weight: bold;
+	text-decoration: none;
+    margin-left: 40px;
+}
+
+.part a:hover {
+	color: grey;
+}
+
+.logo {
+	width: 100px;
+	height:100px;
+	float: left;
+	margin-left: 5%;
+	border: solid 2px white;
+	margin-top: 50px;
+}
+
+section {
+	width: 100%;
+	height: 415px;
+	display: inline;
+}
+
+footer {
+	position: absolute;
+	width: 100%;
+	height: 50px;
+	bottom: 0px;
+	background-color: black;
+	border-top: solid 1px white;
+	opacity: 0.7;
+}
+
+.create-table {
+	height: 270px;
+	width: 19%;
+	border: solid 2px white;
+	margin-left: 5%;
+	margin-top: 8%;
+	background-color: rgba(128,128,128,0.7);
+	color: white;
+	font-weight: bold;
+	text-align: center;
+}
+
+.error {
+	color: red;
+}
+
+.result {
+	margin-top: -350px;
+	margin-left: 500px;
+	width: 55%;
+	height: 500px;
+	border: solid 2px white;
+	background-color: rgba(128,128,128,0.7);
+	visibility: hidden;
+}
+
+.car {
+	height: 250px;
+	width: 100%;
+}
+
+.image {
+	width: 170px;
+	height:170px;
+	margin-left: 150px;
+	margin-top: 30px;
+}
+
+.details {
+	height: 100px;
+	width: 100px;
+	margin-left: 100px;
+}
+
+.edit {
+	margin-left: 100px;
+}
+
+.pic {
+	height: 50px;
+	width: 50px;
+}
```

Comment:
Keep common css in common file as css for footer, header, .create-table are repeated in all css files above